src/factories/smock-contract.ts

```diff
@@ -199,7 +202,8 @@ function parseMessage(message: Message, contractInterface: Interface, sighash: s
   return {
     args: sighash === null ? toHexString(message.data) : getMessageArgs(message.data, contractInterface, sighash),
     nonce: Sandbox.getNextNonce(),
-    target: fromFancyAddress(message.delegatecall ? message._codeAddress : message.to),
+    target: fromFancyAddress(message.delegatecall ? message.codeAddress : message.to),
```

Comment:
why the change in `_codeAddress` vs `codeAddress`?

Reply:
same value actually. Just thought it was nicer